src/instructlab/model/chat.py

```diff
@@ -80,6 +80,14 @@ class ChatException(Exception):
     """An exception raised during chat step."""
 
 
+class ChatFailedToLoadIndexException(ChatException):
+    """An exception raised during chat: RAG is enabled but the index with the RAG content failed to load."""
```

Comment:
`during chat step`? (like the other message)

Reply:
I think "step" is intentionally vague, while this one is more specific.

Reply:
Oh, I see.  Yes, those are kind of inconsistent.  I will change one or the other....